spec/index.html

```diff
@@ -271,7 +271,23 @@ <h2>Notation and Terminology</h2>
     A triple <b><i>appears in</i></b> a graph G
     if it is in G or
     if the triple term composed by its subject, predicate, and object appears in G.
-    <p/>
+    </p>
+
+
+
+
+  <p>
+    For RDF terms t, x, and y, we define the <dfn>substitution mapping</dfn> t[x/y] inductively, as follows:
+  </p>
+  <ol>
+    <li>If t = x, then t[x/y] = y.</li>
+    <li>If t = (s, p, o) and t &ne; x, then t[x/y] = (s[x/y], p[x/y], o[x/y]).</li>
+    <li>Else, t[x/y] = t.</li>
```

Comment:
```suggestion
    <li>If t = x, then t[x/y] = y.</li>
    <li>Otherwise, if t = (s, p, o), then t[x/y] = (s[x/y], p[x/y], o[x/y]).</li>
    <li>Otherwise, t[x/y] = t.</li>
```
I would find this wording slightly clearer, but I won't fight for it.

Reply:
I changed my mind :)